src/App/Pages/Accounts/LoginPageViewModel.cs

```diff
@@ -150,14 +150,24 @@ public async Task InitAsync()
                 {
                     Email = await _stateService.GetRememberedEmailAsync();
                 }
-                var deviceIdentifier = await _appIdService.GetAppIdAsync();
-                IsKnownDevice = await _apiService.GetKnownDeviceAsync(Email, deviceIdentifier);
                 CanRemoveAccount = await _stateService.GetActiveUserEmailAsync() != Email;
-                await _deviceActionService.HideLoadingAsync();
+                IsKnownDevice = await _apiService.GetKnownDeviceAsync(Email, await _appIdService.GetAppIdAsync());
             }
             catch (Exception ex)
             {
-                HandleException(ex);
+                if (ex is ApiException apiException
+                    && apiException.Error.StatusCode == System.Net.HttpStatusCode.Unauthorized)
+                {
+                    _logger.Exception(ex);
+                }
+                else
+                {
+                    HandleException(ex);
+                }
+            }
```

Comment:
IMO this should be as follows to take advantage of the catch, and to have a cleaner version of the code:
```c#
catch (ApiException apiEx) when (apiEx.Error.StatusCode == System.Net.HttpStatusCode.Unauthorized)
{
    _logger.Exception(ex);
}
catch (Exception ex)
{
    HandleException(ex);
}
```